class.jetpack-cli.php

```diff
@@ -906,9 +956,16 @@ public function partner_provision( $args, $named_args ) {
 			}
 		}
 
+		WP_CLI::log( $body_json->next_url );
+
 		WP_CLI::log( json_encode( $body_json ) );
 	}
 
+	private function get_api_host() {
+		$env_api_host = getenv( 'JETPACK_START_API_HOST', true );
+		return $env_api_host ? $env_api_host : JETPACK__WPCOM_JSON_API_HOST;
```

Comment:
I'm not sure how `JETPACK_START_API_HOST` is used, but when it is used, shouldn't it also be used in the `Host` header for requests to work properly?